### DISCUSSION TITLE
What font does pacseek use for its emojis?

### DISCUSSION THREAD

**Discussion by ajgringo619** [moson-mo/pacseek#27] [Q&A]
It seems that every time I need to reinstall my system - yes, I'm a tweaker! - **pacseek's** emojis all become question marks. I always use the same terminal font (`AurulentSansMono Nerd Font`), and always install `noto-fonts-emoji`.

Now, this new install is using **KDE** with the `xfce4-terminal`. My previous (working) systems were using **Xfce** (`xfce4-terminal`), then **Cinnamon** (with `gnome-terminal-transparency`). The **pacseek** config is the same. Can someone explain what I'm doing wrong?

**Comment by moson-mo** [comment]
Make sure you have some fonts installed that have glyphs too.  
For example `noto-fonts`, `ttf-font-awesome`

**Comment by ajgringo619** [comment]
Already had `noto-fonts` installed, adding `ttf-font-awesome` did not help. Here are the fonts I have installed (most of these are defaults from `EndeavourOS`):
```
adobe-source-code-pro-fonts 2.042u+1.062i+1.026vf-1
adobe-source-han-sans-cn-fonts 2.004-1
adobe-source-han-sans-jp-fonts 2.004-1
adobe-source-han-sans-kr-fonts 2.004-1
cantarell-fonts 1:0.303.1-1
fluidsynth 2.3.2-1 (pro-audio)
fontconfig 2:2.14.2-1
freetype2 2.13.0-1
gsfonts 20200910-3
libfontenc 1.1.7-1
libxfont2 2.0.6-2
libxft 2.3.8-1
noto-fonts 20230401-1
noto-fonts-emoji 20220920-1
ttf-bitstream-vera 1.10-15
ttf-dejavu 2.37+18+g9b5d1b2f-3
ttf-font-awesome 6.4.0-1
ttf-liberation 2.1.5-1
woff2 1.0.2-4
xorg-fonts-encodings 1.0.7-1 (xorg-fonts xorg)
```

**Reply by ajgringo619** [reply]
OK, after switching the default font of the Dark Reader extension for Vivaldi to font-awesome, pacseek is now showing the proper emojis again. I'll chalk this up to gremlins.

**Reply by killajoe** [reply]
ttf-font-awesome   is the only one needed here. 
![2023-05-14_08-23](https://github.com/moson-mo/pacseek/assets/16797647/06b9b00e-00ea-496e-8c5e-ec49240c91a4)
Needed only to restart pacseek after installing it to show up indeed..-.
